.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+version: 2.1
+
+commands:
+  runtests:
+    steps:
+      - checkout
+      - run: go get -v -t -d ./...
+      - run: go test -race -v ./...
+
+executors:
+  go18:
+    working_directory: /go/src/github.com/honeycombio/libhoney-go
+    docker:
+      - image: circleci/golang:1.8
+  go19:
+    working_directory: /go/src/github.com/honeycombio/libhoney-go
+    docker:
+      - image: circleci/golang:1.9
+  go110:
+    working_directory: /go/src/github.com/honeycombio/libhoney-go
+    docker:
+      - image: circleci/golang:1.10
+  go111:
+    working_directory: /go/src/github.com/honeycombio/libhoney-go
+    docker:
+      - image: circleci/golang:1.11
+  go112:
+    working_directory: /go/src/github.com/honeycombio/libhoney-go
+    docker:
+      - image: circleci/golang:1.12
+
+jobs:
+  test_go18:
+    executor: go18
+    steps:
+      - runtests
+  test_go19:
+    executor: go19
+    steps:
+      - runtests
+  test_go110:
+    executor: go110
+    steps:
+      - runtests
+  test_go111:
+    executor: go111
+    steps:
+      - runtests
+  test_go112:
+    executor: go112
+    steps:
+      - runtests
+
+workflows:
+  version: 2
```

Comment:
you are using version 2 here because workflows isn't in version 2.1?

Reply:
that was a mistake. it's removed now. the version is supposed to be 2.1 and at the top, not in the workflows section.